### ISSUE TITLE
Requires restart every morning

### ISSUE THREAD

**Issue by jacobgreenberg** [withfig/fig#1014]
### Description:
> Every morning I need to quit and start Fig in order for it to display in terminal. Otherwise, Fig runs but does not appear in terminal. 

### Details:
|macOS|Fig|Shell|
|-|-|-|
|12.2.1|Version 1.0.55 (B383)|/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Version 1.0.55 (B383) [U.S.] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/jacobgreenberg/.fig/bin/fig
Accessibility: true
SSH Integration: false
Tmux Integration: false
iTerm Integration: false 
Hyper Integration: false
VSCode Integration: true
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
UNIX Socket Exists: true
Installation Script: true
PseudoTerminal Path: 
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: ??? (???) - ???
Current terminal session: ???
Current working directory: ???
Current window identifier: ???</pre>
</p>
</details>

**Comment by FabioXimenes** [comment]
I am having the same problem with the same version of Fig in macOS 11.6. This is probably the same problem of #1011, #1012, #1013.

**Comment by brendanfalk** [comment]
So this is the problem:
```
Current active process: ??? (???) - ???
Current terminal session: ???
Current working directory: ???
Current window identifier: ???
```

Have you turned on the setting to hide Fig's icon in the macOS status bar? If yes, I think this may be the cause. Try unhiding it by running

```
 fig settings app.hideMenubarIcon false
```

If this doesn't fix it, we'd love to jump on a call and try and repro. Please let us knwo!

**Comment by FabioXimenes** [comment]
This is my output:

Current active process: zsh (51360) - /dev/ttys006
Current working directory: /Users/vasconfr
Current window identifier: 84154/% (com.apple.Terminal)

I did not hide Fig's icon in the status bar.

**Comment by brendanfalk** [comment]
This issue should likely be fixed as 1.0.56 Build 414. Please run `fig update` and make sure you're on this verison
